config/templates/example.yaml.in

```diff
@@ -38,7 +39,25 @@ spec:
       memory: "8Gi"
   tlsEnabled: true
 {{- /* .LatestStableCrdbVersion: latest stable version of CockroachDB */}}
+# You can set either a version of the db or a specific image name
 # cockroachDBVersion: {{ .LatestStableCrdbVersion }}
   image:
     name: cockroachdb/cockroach:{{ .LatestStableCrdbVersion }}
+  # nodes refers to the number of crdb pods that are created
+  # via the statefulset 
   nodes: 3
+  # affinity is a new API field that is behind a feature gate that is
+  # disabled by default.  To enable please enable, see operator.yaml
+  # The affinity field will accept any podSpec affinity rule.
+  #affinity:
```

Comment:
`#affinity:`

Could you offset this and everything below by 1 space, to be consistent with the rest of the commented blocks?

Reply:
> Could you offset this and everything below by 1 space, to be consistent with the rest of the commented blocks?

If we do this the user will need to delete the `#` and the extra space character as well.  It did it that way so they only delete the `#` character.

Reply:
unfortunately all of our example yaml's are like this so we really should keep to the standard. I would rather change all of them but it'd be an effort to do that and doing it different ways in different files is extremely confusing